q2_feature_classifier/_cutter.py

```diff
@@ -36,6 +36,8 @@ def _primers_to_regex(f_primer, r_primer):
 def _local_aln(primer, sequence):
     best_score = None
     for one_primer in primer.expand_degenerates():
+        # `sequence` may contain degenerates. Although undocumented, SSW will
+        # treat each degenerate character as a mismatch.
```

Comment:
Actually, this comment is specifically about `sequence`, we're still expanding the degenerates in the primer, but we're commenting to purposefully say if there are degenerates in the reference, that we don't care and a mismatch should still be "good enough".